test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchPersistenceIntegrationTest.java

```diff
@@ -150,6 +166,46 @@ public void testWorkbenchProjectPersistence() {
         assertThat(containersResponse.getResult().getContainers().get(0).getContainerId()).isEqualTo(CONTAINER_ID);
     }
 
+    @Test
+    public void testGitHooksPersistence() {
+        // get git hook dir location
+        String gitHooksRemoteDir = "/opt/eap/standalone/data/bpmsuite/git/hooks"; //TODO for 7.1 bpmsuite -> kie
+
+        updateWorkbenchDeployment(gitHooksRemoteDir);
+
+        logger.debug("Filter instances for second deployment");
+        List<String> workbenchInstanceNames = workbenchDeployment.getInstances().stream().map(Instance::getName).filter(name -> name.contains("-2-")).collect(Collectors.toList());
```

Comment:
What is the purpose of this filtering? Does it mean that getInstances() returns also pods which are not active?

Reply:
To make sure, that deployment was redeployed and we are working with updated pod. There is small chance, we can get active pod, where is undeploying previous configuration. But I think that filtering can be removed for now.